src/Startup.cs

```diff
@@ -47,5 +55,21 @@
                 context.Response.StatusCode = 404;
             }
         });
+    }
+
+    // This method configures OpenTelemetry tracing.
+    public void ConfigureOpenTelemetryTracing()
+    {
+        var tracerProvider = Sdk.CreateTracerProviderBuilder()
+            .AddAspNetCoreInstrumentation()
+            .AddSource(EndpointBase.ActivitySourceName)
+            .SetResourceBuilder(ResourceBuilder.CreateDefault()
+                .AddService(Program.OpcPlcServer.Config.ProgramName))
+            .AddOtlpExporter(opt => {
+                opt.Endpoint = new Uri("http://otel-collector.opcuabroker-monitoring.svc.cluster.local:4317");
```

Comment:
>Uri [](http://example.com/codeflow?start=35&length=3)

Shouldn't this uri come from the config?

Reply:
yes ,removed debug code